noaagen.py

```diff
@@ -15,9 +16,10 @@ def print_hi(name):
     file = open('/surf/gendata.txt')
     content = file.readlines()
     data = content[0]+content[1]+content[2]
-    data = re.sub(tlreg, '', data)
-    data = re.sub(mlreg, '', data)
-    data = re.sub(blreg, '', data)
+    timestamp = datetime.now().strftime("%m/%d/%y %H:%M")
+    data = re.sub(tlreg, '  TIMESTAMP', data)
+    data = re.sub(mlreg, '  MM/DD/YY hh:mm TZ', data)
+    data = re.sub(blreg, f'  {timestamp} PST', data)  # datetime.now() returns local time, so assume PST
```

Comment:
Note: I am hardcoding `PST` here with the assumption that the host is running in the PST timezone. There is a potential to use `pytz` but that is out of scope for the Python Standard Library and will require a pip install on the host.